src/platform/posix/posix_epdesc.c

```diff
@@ -131,9 +131,9 @@ nni_posix_epdesc_doaccept(nni_posix_epdesc *ed)
 	int      newfd;
 
 	while ((aio = nni_list_first(&ed->acceptq)) != NULL) {
-// We could argue that knowing the remote peer address would
-// be nice.  But frankly if someone wants it, they can just
-// do getpeername().
+		// We could argue that knowing the remote peer address would
+		// be nice.  But frankly if someone wants it, they can just
+		// do getpeername().
```

Comment:
This goofy indentation (my old one I mean) may have been done automatically by clang-format.  In which case it may wind up getting reverted later.  I don't always agree 100% with the decisions it makes, but I find using it consistently means I don't have to get into debates about formatting style.

Still it isn't as aggressive about undoing changes as some other formatting tools, so its possible that this change won't get trashed.   This is just a heads up in case it does.

Reply:
Looking at this more closely, we should remove this comment ... its no longer relevant due to other changes in the code.